src/panels/config/devices/device-detail/ha-device-info-card.ts

```diff
@@ -135,6 +138,10 @@ export class HaDeviceCard extends LitElement {
         color: var(--secondary-text-color);
         word-wrap: break-word;
       }
+      a {
+        text-decoration: underline;
+        color: var(--primary-color);
+      }
```

Comment:
```suggestion
```

Reply:
Use `haStyle`;

```
  static get styles(): CSSResultGroup {
    return [haStyle, css`
      :host {
        display: block;
      }
      ....
      `,
   ];
```

Reply:
This is needed, or else the link will be default dark blue.

Reply:
the text-decoration can go actually, I experimented doing it without, but that looked of as well.

Reply:
I now see your latest comment, had the tab open, don't know why GitHub didn't add that response (reload page revealed it). Will adjust.